template/src/main/twirl/ch.epfl.scala.index.views/main.scala.html

```diff
@@ -163,5 +163,8 @@ <h5>Follow us:</h5></li>
 
       Raven.config('https://d9ae900df7ef420b86063800d541d712@@sentry.io/185020').install()
     </script>
+    <script type="text/javascript">
+        addCopyListenersOnClass("btn-copy");
```

Comment:
What about calling this from the `Client.main` entry point instead of having two entry points?